contrib/ohi-release/action.yml

```diff
@@ -0,0 +1,74 @@
+name: OHI release
+description: Wrapper for release toolkit that runs commands needed to release an OHI
+inputs:
+  excluded-dirs:
+    description: 'Exclude commits whose changes only impact files in specified dirs relative to repository root. Defaults to ".github".'
+    required: true
```

Comment:
I would leave all this parameters as required false.

Reply:
Solved in [#b75bd12](https://github.com/newrelic/release-toolkit/pull/135/commits/b75bd12b5be2dc02d4f5aae737ed14a9efc90c6f)